common/app/common/NewNavigation.scala

```diff
@@ -535,9 +366,67 @@ object NewNavigation {
     )
 
     def getTertiaryNavLinks(frontId: String): List[TertiaryLink] = {
-      tertiaryLinks.filter{ item =>
+      tertiaryLinks.filter { item =>
         item.frontId == frontId
       }
     }
   }
+
+  object SectionLinks {
+
+    var sectionLinks = List(
+
+      SectionsLink("uk", News),
```

Comment:
Feels like these could be represented differently to avoid the repetition but maybe as a future PR (because I don't know how off the top of my head!) :)